gpkit/nomials/variables.py

```diff
@@ -148,6 +148,8 @@ def __new__(cls, shape, *args, **descr):
             i = it.multi_index
             it.iternext()
             descr.update({"idx": i})
+            assert value_option is not None
+            # TODO refactor logic for value_option and remove above assert
```

Comment:
for the record, this commit is by @whoburg; it's an amended version of 419ff866eda5e861266182c2177ef30e919442f8